docs/guides/usage/tracing.livemd

```diff
@@ -111,8 +148,9 @@ Integer.parse("1")
 <!-- livebook:{"output":true} -->
 
 ```
-Matcha.Trace: `Elixir.Integer.parse("1")` called on #PID<0.142.0>
```

Comment:
The `Integer.__info__(:deprecated)` stuff is happening for some reason I don't understand; it wouldn't get called in normal code execution and spoils the examples. I think it's some iex-mode-only introspection thing?